install_hotfix.py

```
@@ -0,0 +1,256 @@
+            logger.info(f"✅ Pydantic {version} detected on {platform_info['system']}")
+            
+        return True
+            
+    except ImportError:
+        logger.warning(f"⚠️  Pydantic not found. Installing...")
+        
+        if platform_info['is_fedora']:
+            # Try system package first on Fedora
+            logger.info("🐧 Attempting Fedora system package installation...")
+            try:
+                subprocess.check_call(['sudo', 'dnf', 'install', '-y', 'python3-pydantic'], 
```

Ruff (0.11.9)

Trailing whitespace

Remove trailing whitespace

(W291)